client-csharp/utils/DataStructure/RowRecord.cs

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+using System.Collections.Generic;
+using Thrift;
+using System;
+namespace iotdb_client_csharp.client.utils
+{
+    public class RowRecord
+    {
+        public long timestamp{get;set;}
+        public List<Object> values {get;set;}
+        public List<string> measurements{get;set;}
+        public RowRecord(long timestamp, List<Object> values, List<string> measurements){
+            this.timestamp = timestamp;
+            this.values = values;
+            this.measurements = measurements;
+        }
+        public void append(string measurement, Object value){
+            values.Add(value);
+            measurements.Add(measurement);
+        }
+        public DateTime get_date_time(){
+            return DateTimeOffset.FromUnixTimeMilliseconds(timestamp).DateTime.ToLocalTime();;
+        }
+
+        public override string ToString()
+        {
+            var str = "TimeStamp";
+             foreach(var measurement in measurements){
+                str += "\t\t";
+                str += measurement.ToString();
+            }
+            str += "\n";
+
+            str += timestamp.ToString();
+            foreach(var row_value in values){
+                str += "\t\t";
+                str += row_value.ToString();
+            }
+            return str;
+        }
+        public List<int> get_datatypes(){
```

Comment:
Can you use interfaces instead of class type? In this case, better to return ``IReadOnlyCollection`` or ``ICollection`` instead of ``List<int>``